src/platform/Zephyr/ConfigurationManagerImpl.cpp

```diff
@@ -210,6 +214,22 @@ void ConfigurationManagerImpl::DoFactoryReset(intptr_t arg)
     ConnectivityMgr().ErasePersistentInfo();
 #endif
 
+#ifdef CONFIG_CHIP_FACTORY_RESET_ERASE_PSA_ITS
+    // Ensure that all persistent PSA crypto materials are removed.
+    for (uint32_t keyID = static_cast<uint32_t>(chip::Crypto::KeyIdBase::Minimum);
```

Comment:
```suggestion
    for (psa_key_id_t keyID = chip::to_underlying(Crypto::KeyIdBase::Minimum);
```
etc. to not cast _via_ `uint32_t`.

Though now that I think about it, this will invoke `psa_destroy_key` for 255 possible fabrics even though we have only 5 configured. Maybe it's better to run this loop only for the [minimum, minimum + MAX_FABRICS) range? Or it doesn't generate any latency?